APE0.rst

```diff
@@ -0,0 +1,198 @@
+The Coordination Committee should look for ways to use these powers as little as possible, striving to involve the community whenever possible for decisions, and delegating powers.
+
+Making decisions
+^^^^^^^^^^^^^^^^
+To use its powers, if consensus cannot be reached within the committee, the Coordination Committee votes. Every Coordination Committee member must either vote or explicitly abstain. Members with conflicts of interest on a particular vote must abstain. The final decision regarding what constitutes a conflict of interest in terms of serving on the Coordination Committee, during elections or individual votes is determined by the Ombudsperson in accordance with a published conflict of interest policy. Passing decisions requires a strict majority of non-abstaining Coordination Committee members.
+Whenever possible, the Coordination Committee's deliberations and votes shall be held in public in a manner which permits discussion and comments from the community.
+
+Electing the Coordination Committee
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+A Coordination Committee election is held annually to fill any vacant seats. Notice of the election must be posted via the Project’s standard communication channels (reaching all active Voting Members) at least one month before the beginning of Phase 1.  The election consists of two phases, each lasting two weeks:
+* Phase 1: Candidates advertise their interest in serving. Candidates can be nominated by anyone, including self-nominations, and do not have to be Voting Members themselves. 
+* Phase 2: Each voting member can vote for zero or more of the candidates, up to the number of candidates. Voting is performed anonymously. Candidates are ranked by the total number of votes they receive. If a tie occurs, it may be resolved by mutual agreement among the candidates, or else the winner will be chosen at random.
+The election process is managed by a non-voting Returns Officer nominated by the outgoing Coordination Committee. For the initial election, the Returns Officer will be nominated by the NumFOCUS Executive officer or their delegate. Elections will be carried out every year, provided that at least one of the Coordination Committee members has reached the end of their term or decided to step down.
+
+Term
+^^^^
+Each Coordination Committee member's term runs for three years from when the election results are finalized, except when the election was to fill a vacancy created part way through a term. In that case the newly-elected member’s term runs for the remainder of the term of the person they are replacing. There are no limits on the number of terms that a single individual can be elected for.
+For the initial election of Coordination Committee members, two seats will have terms of one year, one has a term of two years and one has a term of three years so as to create a staggered set of replacements and provide continuity in the Coordination Committee.
```

Comment:
How will it be decided who stays on how long? Based on votes received? (if so, specify)

Reply:
Hmm. This is a sticky problem.  I'll re-word to address this.